reports/California.md

```diff
@@ -718,7 +782,7 @@ id: ca-sanjose-5
 * https://www.youtube.com/watch?v=89mUHzu3480
 
 
-### Police strike man with motorcylce | May 31st
+### Police strike man with motorcycle | May 31st
 
 A man is seen run across a street, chased by several officers. An officer on a motorcycle strikes him and he is arrested.
```

Comment:
should probably change this to `running` while making updates.  Idk if that comma is needed either